jihyeon/routes/userRoute.js

```diff
@@ -4,6 +4,7 @@ const userController = require("../controllers/userController");
 const router = express.Router();
 
 router.post("/signup", userController.signUp);
+router.get("/login", userController.login);
```

Comment:
```suggestion
router.post("/login", userController.login);
```
login 정보를 받을 때는 body를 통해 받으므로 get이 아닌 post 메소드를 통해 처리되어야합니다.